src/PKSim.Presentation/Presenters/Individuals/IndividualTransporterExpressionsPresenter.cs

```diff
@@ -99,6 +104,22 @@ public void ActivateMolecule(IndividualMolecule molecule)
          rebind();
       }
 
+      private void updateWarning()
+      {
+         if (_transporterExpressionDTO.DefaultAvailableInDatabase)
```

Comment:
so here is what I am doing. In the DTO I save if this came from DB or not. If yet, then nothing to show
If not, then I check if the transporter type is our default, (Efflux). If yes, I show the warning. Otherwise hide
This means that this warning will always be shown for transporter not from db and Efflux. Are we okay with this?
Otherwise, we need to save the info that the value was set by the user as an extra prop in the transporter. Doable
Thoughts?